ui/client/actions/nk/toolbars.ts

```diff
@@ -0,0 +1,58 @@
+import {ToolbarsSide} from "../../reducers/toolbars"
+import {ThunkAction} from "../reduxTypes"
+import {RootState} from "../../reducers/index"
+import {Toolbar} from "../../components/toolbarComponents/toolbar"
+
+export type ToolbarPosition = [ToolbarsSide | string, number]
+
+type ResetToolbarsAction = { type: "RESET_TOOLBARS", toolbars: Array<[string, ToolbarsSide]> }
+type RegisterToolbarsAction = { type: "REGISTER_TOOLBARS", toolbars: Array<[string, ToolbarsSide]> }
+type MoveToolbarAction = { type: "MOVE_TOOLBAR", from: ToolbarPosition, to: ToolbarPosition }
+type ToggleToolbarAction = { type: "TOGGLE_TOOLBAR", id: string, isCollapsed: boolean }
+type ToggleToolboxGroupAction = { type: "TOGGLE_NODE_TOOLBOX_GROUP", nodeGroup: string }
+type ToggleAllToolbarsAction = { type: "TOGGLE_ALL_TOOLBARS", isCollapsed: boolean }
+
+export const toggleAllToolbars = (isCollapsed: boolean): ToggleAllToolbarsAction => ({type: "TOGGLE_ALL_TOOLBARS", isCollapsed})
+
+export const resetToolbars = (): ThunkAction<RootState> => {
+  return (dispatch, getState) => {
+    const toolbars = getState().toolbars?.initData
+    dispatch({type: "RESET_TOOLBARS", toolbars})
+  }
+}
+
+export function registerToolbars(toolbars: Toolbar[]): RegisterToolbarsAction {
+  return {
+    type: "REGISTER_TOOLBARS",
+    toolbars: toolbars.map(({id, defaultSide}) => [id, defaultSide]),
+  }
+}
+
+export function moveToolbar(from: ToolbarPosition, to: ToolbarPosition): MoveToolbarAction {
+  return {
+    type: "MOVE_TOOLBAR",
+    from, to,
+  }
+}
+
+export function toggleToolbar(id: string, isCollapsed = false): ToggleToolbarAction {
+  return {
+    type: "TOGGLE_TOOLBAR",
+    id, isCollapsed,
```

Comment:
why not on separate lines?

Reply:
because there is no such rule so I don't care and I do what is comfortable for me.  I think that place for imposed formatting rules is in .eslintrc not in PRs.